sympy/simplify/tests/test_radsimp.py

```diff
@@ -393,6 +394,8 @@ def test_fraction():
     assert fraction(exp(n)) == (1, exp(-n))
     assert fraction(exp(-n)) == (exp(-n), 1)
 
+    p = symbols('p', positive=True)
+    assert fraction(exp(-p)*log(p), exact=True) == (exp(-p)*log(p), 1)
```

Comment:
> I referred to the docstring while making this PR. Since the docstring only refers to the case when exact is unset. It doesnt talk about if exact=True.

OK we have four cases:

1. exact and vanilla (sign unknown) exponent -- your case
2. not exact and vanilla -- docstring case
3. exact and not vanilla - not covered by docstring; should `exact` take precedence?
4. not exact and not vanilla - pretty result